pyiron_base/jobs/job/runfunction.py

```diff
@@ -617,7 +617,7 @@ def run_job_with_runmode_executor_flux(job, executor, gpus_per_slot=None):
     )
     jobspec.cwd = job.project_hdf5.working_directory
     jobspec.environment = dict(os.environ)
-    return executor.submit(jobspec)
+    job.server.future = executor.submit(jobspec)
```

Comment:
I didn't follow everything in detail, but are we not doing the `fs = job.run()` anymore?  Is that taken of somewhere else or is it not needed anymore?

Reply:
Yes, we do not do it anymore. We decided to put the future next to the executor onto the server object. As such you cannot forget to assign the run call. Especially since we never did this before..